src/Integration/NewConnectedMode/IConfigurationProvider.cs

```diff
@@ -30,6 +30,6 @@ internal interface IConfigurationProvider : ILocalService
         /// </summary>
         BoundSonarQubeProject GetBoundProject();
 
-//        ConnectedMode GetMode();
+        SonarLintMode GetMode();
```

Comment:
The interface now returns the mode (although the implementation only supports standalone mode and legacy connected mode currently)